torch/onnx/utils.py

```diff
@@ -1314,6 +1314,71 @@ def _validate_dynamic_axes(dynamic_axes, model, input_names, output_names):
             dynamic_axes[key] = value_dict
 
 
+# This function export torch::jit::Graph object
```

Comment:
Do we still keep these commented out functions?

Reply:
These are intended, showing how this api will be invoked.